pkg/ui/workspaces/cluster-ui/src/store/databasesList/databasesList.selectors.ts

```diff
@@ -9,9 +9,25 @@
 // licenses/APL.txt.
 
 import { createSelector } from "reselect";
-import { adminUISelector } from "../utils/selectors";
+import { adminUISelector, localStorageSelector } from "../utils/selectors";
+import { LocalStorageKeys } from "../localStorage";
 
 export const databasesListSelector = createSelector(
   adminUISelector,
   adminUiState => adminUiState.databasesList,
 );
+
+export const selectDatabasesSortSetting = createSelector(
+  localStorageSelector,
+  localStorage => localStorage[LocalStorageKeys.DB_SORT],
+);
+
+export const selectDatabasesFilters = createSelector(
+  localStorageSelector,
+  localStorage => localStorage["filters/DatabasesPage"],
```

Comment:
nit: same with the keys here and below.
Also my 2 cents is I don't think it's worthwhile using `createSelector` on these as you're just reading fields on an object.

Reply:
Added storage keys, selectors are no longer memoized.